src/ParallelAlgorithms/LinearSolver/Multigrid/Tags.hpp

```diff
@@ -43,6 +43,34 @@ struct OutputVolumeData {
   static bool suggested_value() { return false; }
 };
 
+template <typename OptionsGroup>
+struct EnablePreSmoothing {
+  static std::string name() { return "PreSmoothing"; }
+  using type = bool;
+  static constexpr Options::String help =
+      "Set to 'False' to disable pre-smoothing altogether (\"cascading "
+      "multigrid\"). Note that pre-smoothing can be necessary to remove "
+      "high-frequency modes in the data that get restricted to coarser grids, "
```

Comment:
in the data that get(s) ? I assume the data gets restricted and not the high-frequency modes in this sentence.

Reply:
either way I think `get` is correct because `data` is always plural 🤓